lib/eclipse/include/opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp

```diff
@@ -111,6 +111,11 @@ namespace Opm
         const MessageContainer& getMessageContainer() const;
         MessageContainer& getMessageContainer();
 
+        /*
+          Will remove all completions which are connected to cell which is not
+          active. Will scan through all wells and all timesteps.
+        */
+        void filter_completions(const EclipseGrid& grid);
```

Comment:
Is there a reason whyyou call it filter_completions() rather than filterCompletions()?

Reply:
Change of opinion; I have come to like `filter_completions`better - but maybe that is too late. I guess I should should change it to `filterCompletions()`?

Reply:
Since that is the convention followed for the rest of opm-parser (and indeed most of the rest of OPM) I think we should keep it.

I used to use camelCaseMethods and underscore_data_members, but have nearly been convinced to camelCaseEverything since that is used by most of the other developers. So I was a bit surprised to see your opinion swinging the other way again... (although for methods).

Reply:
I will change.